src/components/_common/overview-filters/OverviewFilters.tsx

```diff
@@ -78,8 +77,13 @@ const DesktopView = ({
     showAreaFilter,
     showTaxonomyFilter,
 }: Props) => {
+    const { language } = usePageConfig();
+    const searchLabel = translator('overview', language)('filterOrSearch');
     return (
         <div className={style.desktop}>
+            <Heading className="sr-only" level={'2'} size={'xsmall'}>
```

Comment:
Finner ikke dette klassenavnet i CSS'en. Kommer det fra ds-css?

Reply:
Det stemmer. Det er en klasse som skjuler noe visuelt, men fortsatt beholder det tilgjengelig for skjermleser.